src/main/java/com/minecolonies/entity/ai/citizen/farmer/EntityAIWorkFarmer.java

```diff
@@ -199,17 +217,95 @@ private boolean canGoPlanting(@NotNull Field currentField, @NotNull BuildingFarm
 
         if (shouldTryToGetSeed)
         {
+            int slot = worker.findFirstSlotInInventoryWith(currentField.getSeed());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'slot' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)